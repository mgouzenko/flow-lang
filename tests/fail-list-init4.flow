// Should not be able to init list with list of differnt type

int main() {
  list <int> test1 = [2, 3];
  list <double> test2 = test1;
}