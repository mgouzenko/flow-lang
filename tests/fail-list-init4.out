Fatal error: exception Failure("test2: Declaration type does not match expression")